app/views/admin/accounting/transactions/_form.slim

```diff
@@ -30,6 +30,13 @@
       - if @transaction.customer
         = @transaction.customer.name
 
+  = f.input :qb_department
+    .form-element
+      = (@transaction.qb_department ? @transaction.qb_department.name : t('quickbooks.department_not_set'))
```

Comment:
I'm not seeing a "department not set" message in the translation files, and the message in the screenshots no longer aligns with the behavior we want (prevent creating txns without a department set instead of allowing them with a blank department)

Reply:
I see the message now, it just wasn't flagged as a change, my bad - and I see now that 11046 is probably the one that's going to change the message/behavior related to creating txns. Ignore me.

Reply:
yes - that is correct, 11046 has the change to require dept to make a txn. Thanks for keeping eye out!